pkgs/applications/misc/mapproxy/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "MapProxy";
+  version = "1.12.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "622e3a7796ef861ba21e42231b49c18d00d75f03eaf3f01a2b7687be7568e2ec";
+  };
+
+  prePatch = ''
+    substituteInPlace mapproxy/util/ext/serving.py --replace "args = [sys.executable] + sys.argv" "args = sys.argv"
+  '';
+
+  buildInputs = [
+    pillow
+    pyyaml
+    python3Packages.pyproj
+    shapely
+    gdal
+    lxml
+  ];
+
+  doCheck = false;
```

Comment:
why disabled?

Reply:
Hi @FRidh ,
As mentioned above https://github.com/NixOS/nixpkgs/pull/68628#issuecomment-531134364

The test cases are disabled because 
Dependency list is huge.
https://github.com/mapproxy/mapproxy/blob/master/requirements-tests.txt
There are security issues with package Riak
https://github.com/NixOS/nixpkgs/issues/33876
https://github.com/NixOS/nixpkgs/pull/56480

Reply:
explain that with a comment in the code